packages/front-end/components/EventWebHooks/EventWebHooksPage.tsx

```diff
@@ -1,23 +0,0 @@
-import React from "react";
-import usePermissions from "@/hooks/usePermissions";
-import { EventWebHookListContainer } from "./EventWebHookList/EventWebHookList";
-
-export const EventWebHooksPage = () => {
-  const permissions = usePermissions();
-
-  if (!permissions.manageWebhooks) {
-    return (
-      <div className="pagecontents">
-        <div className="alert alert-danger">
-          You do not have access to view this page.
-        </div>
-      </div>
-    );
-  }
-
-  return (
-    <div className="pagecontents">
-      <EventWebHookListContainer />
-    </div>
-  );
-};
```

Comment:
We were checking the user's permission level twice - once here, and one on the page this component's parent component. Removing it make this abstraction unnecessary, imo.